login-workflow/docs/services.md

```diff
@@ -50,6 +50,12 @@ constructor(pxbAuthConfig: PxbAuthConfig) {
 -   **customEmailValidator** (optional): _`ValidatorFn`_
     -   Custom email regex requirements.
     -   Default: `undefined`
+-   **customFirstNameRequirements** (optional): _`NameRequirement[]`_
+    -   Array of custom first name requirements.
+    -   Default: `[]`
+-   **customLastNameRequirements** (optional): _`NameRequirement[]`_
+    -   Array of custom last name requirements.
+    -   Default: `[]`
```

Comment:
Is there documentation for the `NameRequirement` type? 

From the docs I see here, I wouldn't know how to implement this. Although it's out of scope from this story and it's been there for some time, `PasswordRequirement` would be the same case.

Reply:
I would recommend doing this a similar way to how we did for R/RN — using a single configuration property for everything so we can avoid prop explosion if we want to add new fields later:
https://github.com/pxblue/react-auth-shared/blob/dev/docs/API.md#registrationconfig

Reply:
I like the idea of having a regex option, but I don't know if I would rely exclusively on that (kind of annoying for a user to have to set up a regex just to do a character limit). We should actually be preventing users from entering more than the max allowed rather than showing an error message, imo

Reply:
I've just ripped out the regex customization since it's overkill for just enforcing a length restriction.  Now this workflow API should match pretty close against RN.  

In the future if we need more advanced customization, then we can always add regex customization back in.